Source/WebKit/NetworkProcess/NetworkProcess.h

```diff
@@ -296,6 +296,7 @@ class NetworkProcess : public AuxiliaryProcess, private DownloadManager::Client,
     void setBackupExclusionPeriodForTesting(PAL::SessionID, Seconds, CompletionHandler<void()>&&);
 #endif
     void clearStorage(PAL::SessionID, CompletionHandler<void()>&&);
+    void cloneStorageForWebPage(PAL::SessionID, WebPageProxyIdentifier, WebPageProxyIdentifier);
```

Comment:
We shouldn't omit the names of the last two parameters since they are not obvious.